CONTRIBUTING.md

```diff
@@ -223,6 +223,18 @@ See [`apm-agent-core/README.md`](apm-agent-core/README.md)
 
 See [`apm-agent-plugins/README.md`](apm-agent-plugins/README.md)
 
+### Documenting
+
+HTML Documentation is generated from text files stored in `docs` folder using [AsciiDoc](http://asciidoc.org/) format.
```

Comment:
```suggestion
[HTML Documentation](https://www.elastic.co/guide/en/apm/agent/java/current/index.html) is generated from text files stored in `docs` folder using [AsciiDoc](http://asciidoc.org/) format.
```

Reply:
We actually use [Asciidoctor](https://asciidoctor.org/) now, not Asciidoc.